docs/db/changelog/changesets/powerauth-nextstep/1.4.x/20230324-init-db.xml

```diff
@@ -10,7 +10,7 @@
             </not>
         </preConditions>
         <comment>Create a new sequence ns_operation_afs_seq</comment>
-        <createSequence sequenceName="ns_operation_afs_seq" cacheSize="20" />
+        <createSequence sequenceName="ns_operation_afs_seq" startValue="1" incrementBy="1" cacheSize="20" minValue="1" />
```

Comment:
We set `minValue` here however not in other sections. We should probably always se this.

Reply:
Ah, I see there is already the `startValue`, so this setting seems rather redundant. Anyway it is not consistent.

Reply:
Apologies, this was forgotten by mistake